docs/controls/PairedResult.md

```diff
@@ -108,16 +108,35 @@ Refer [Graph](../core/Graph.md) `Root` for more details.
 | key           | string   | Unique id which represents the data-set |
 | values        | Array    | [Values](#values)                       |
 
+#### Optional 
+
+| Property Name | Expected | Default            | Description                                                                       |
+| ------------- | -------- | ------------------ | ----------------------------------------------------------------------------------|
+| yAxis         | string   | "y"                | Setting for using different Y based axis. For now: its either Y or Y2             |
+| regions       | object   | {}                 | Refer [Regions](#regions)                                                         |
+| label         | object   | {}                 | Display value for the data-set which the data points belong to                    |
+| color         | string   | COLORS.BLACK       | Color for the data point                                                          |
+| shape         | string   | SHAPES.DARK.CIRCLE | Shape for representing the data points                                            |
+| onClick       | Function | undefined          | Any action that can be performed when clicking on the data point                  |
+| legendOptions | object   | undefined          | Option to show/hide shape and/or line. Refer [LegendOption(#legendOptions)        |
+
+##### LegendOptions
+
+Each line can have a legendOptions object in [Values](#values) level.
+
+###### Required
+
+| Property Name | Expected | Default | Description                                           |
+| ------------- | -------- | ------- | ----------------------------------------------------- |
+| showLine      | boolean  | false   | Display Line in the legend.                           |
+| showShape     | boolean  | true    | Display Shape in the legend.                          |
+| style         | object   | {}      | Any style that can be applied. Refer [Styles](#style) |
```

Comment:
Styles link also not working

Reply:
Fixed [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/0c528924d29c82332690c3ec3e7d77aa180c8b9a#diff-ac5dee9f2f5653ec431590b47bc921f2R141)